src/Common/tests/System/PlatformDetection.cs

```diff
@@ -31,8 +31,10 @@ public static partial class PlatformDetection
         public static bool IsNetBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
         public static bool IsOpenSUSE => IsDistroAndVersion("opensuse");
         public static bool IsUbuntu => IsDistroAndVersion("ubuntu");
-        public static bool IsWindowsNanoServer => (IsWindows && !File.Exists(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Windows), "regedit.exe")));
+        public static bool IsWindowsNanoServer => (IsWindows && IsNotWindowsIoTCore && !File.Exists(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Windows), "regedit.exe")));
```

Comment:
Right, there is no GDI.  IoT still supports displays but it's supported by DWM directly, so headless isn't really accurate.